DigitalLearningSolutions.Web/Helpers/TrackingSystem/CentrePage.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers.TrackingSystem
+{
+    public enum CentrePage
+    {
+        CentreDashboard,
+        CentreConfiguration,
+        CentreAdministrators,
+        Reports

```

Comment:
Should these perhaps be named Dashboard/Configuration/Administrators/Reports i.e. drop the Centre from all of them since that is clear from the enum name (and all our urls will ultimately be /centre/pagename)